MonoGame.Framework.Content.Pipeline/Processors/FontDescriptionProcessor.cs

```diff
@@ -53,6 +56,8 @@ public override SpriteFontContent Process(FontDescription input, ContentProcesso
                         if (File.Exists(fontFile))
                             break;
                     }
+                    if (File.Exists(fontFile))
```

Comment:
Full code 

`foreach (var dir in directories)
                {
                    foreach(var ext in extensions)
                    {
                        fontFile = Path.Combine(dir, input.FontName + ext);
                        if (File.Exists(fontFile))
                            break;
                    }
                    if (File.Exists(fontFile)) // We need this to break out of the `directories` for each
                        break;
                }`